### ISSUE TITLE
ERROR: zipfile.LargeZipFile: Files count would require ZIP64 extensions

### ISSUE THREAD

**Issue by MurphyDavid** [gaow/SEQLinkage#10]
Hi

We've been getting a slightly odd error while trying out SEQLinkage.

This is the command we're using:

> docker run -v $PWD:/home/seqx/data gaow/seqx seqlink --vcf dHMN5_family.vcf.gz --fam FAM.txt --output Test --run-linkage --prevalence 0.000001 --moi AD --theta-max 0.5 --wt-pen 0.00 --mut-pen 1.00 --jobs 8 --output-entries 10

dHMN5_family.vcf.gz is a vcf file for 6 individuals whole exomes and is ~20MB zipped, 77MB unzipped. 

We get an error with stack trace as bellow. 

> [1;40;32mMESSAGE: 19,382 units successfully converted to LINKAGE format[0m
> [1;40;32mMESSAGE: Archiving LINKAGE format to directory [/home/seqx/data/cache][0m
> Traceback (most recent call last):
>   File "/opt/miniconda3/envs/snake/bin/seqlink", line 4, in <module>
>     __import__('pkg_resources').run_script('SEQLinkage==1.0.0', 'seqlink')
>   File "/opt/miniconda3/envs/snake/lib/python2.7/site-packages/setuptools-20.6.7-py2.7.egg/pkg_resources/__init__.py", line 719, in run_script
>   File "/opt/miniconda3/envs/snake/lib/python2.7/site-packages/setuptools-20.6.7-py2.7.egg/pkg_resources/__init__.py", line 1490, in run_script
>   File "/opt/miniconda3/envs/snake/lib/python2.7/site-packages/SEQLinkage-1.0.0-py2.7.egg/EGG-INFO/scripts/seqlink", line 103, in <module>
>     
>   File "/opt/miniconda3/envs/snake/lib/python2.7/site-packages/SEQLinkage-1.0.0-py2.7.egg/SEQLinkage/Core.py", line 690, in main
>   File "/opt/miniconda3/envs/snake/lib/python2.7/site-packages/SEQLinkage-1.0.0-py2.7.egg/SEQLinkage/Utils.py", line 584, in write
>   File "/opt/miniconda3/envs/snake/lib/python2.7/site-packages/SEQLinkage-1.0.0-py2.7.egg/SEQLinkage/Utils.py", line 351, in zipdir
>   File "/opt/miniconda3/envs/snake/lib/python2.7/zipfile.py", line 1148, in write
>     self._writecheck(zinfo)
>   File "/opt/miniconda3/envs/snake/lib/python2.7/zipfile.py", line 1114, in _writecheck
>     " would require ZIP64 extensions")
> zipfile.LargeZipFile: Files count would require ZIP64 extensions

Looking up the error it looks like it may be related to large numbers of small files being created in one folder but we're not sure if it's a mistake on our part or just an odd bug. 

Kind regards
David Murphy

**Comment by PsoAE** [comment]
Hi,
I have the same problem, but can not solve it. I read in other forums that "allowZip64 = True" has to be added to the python scripts. Actually the number of files and the amount of data is expected to be large when using vcf files (i. e. NGS data), so I am wondering why this should be a problem.

Regards!

**Comment by gaow** [comment]
It is a bit hard to debug since I cannot reproduce the problem (without a large VCF file in hand). But my `zipdir` code does have `allowZip64 = True`. I found [another line in the code](https://github.com/gaow/SEQLinkage/blob/81c37bbc9cc62fdff0debd0b78bd7e8f006bc10a/source/Utils.py#L569) that possibly needs this option so I added it, though I doubt this is where it raises the problem.

I've updated the docker image. Notice that the command to run the new docker image is slightly different from the old image. See [here](http://bioinformatics.org/seqlink/installation) for the updated command.
